hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,11 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  Kafo::Helpers.execute('rpm -e elasticsearch lucene3 ruby193-rubygem-tire cigar') if system('rpm -q elasticsearch')
+  Kafo::Helpers.execute('rm -rf /var/lib/elasticsearch')
```

Comment:
I'm kinda hesitant to blindly rm -rf this directory.  I'd rather we document that the user can do this.